.env.default

```diff
@@ -0,0 +1 @@
+CISCOSPARK_APPID_ORGID=Y2lzY29zcGFyazovL3VzL09SR0FOSVpBVElPTi80MjFhMWY4OC0zM2ViLTQ3MTUtOTY1ZS0zYzE5ZDYyZDJjMDk
```

Comment:
Should we keep the `APPID_ORGID` static here? I think it makes more sense to store both in the Jenkins config since one is useless without the other. If we need to change it, it would force us to do a PR instead of only making the change in Jenkins... unless that's the reason for splitting them up

Reply:
I'm simply matching the way the sdk utilizes the app id org field. We may need to change these in both places once @mattnorris's work is done